EXOSIMS/Prototypes/TargetList.py

```diff
@@ -342,6 +349,29 @@ def __init__(
         self.earths_only = bool(earths_only)
         self.scaleWAdMag = bool(scaleWAdMag)
         self.skipSaturationCalcs = bool(skipSaturationCalcs)
+        try:
+            massLuminosityRelationship = specs["modules"]["massLuminosityRelationship"]

```

Comment:
this keyword should be in the top-level of the input script, not under module. you should not need a try/except - the default should come from the default value in the `__init__` declaration (which appears to be missing).